queries/fidl/highlights.scm

```diff
@@ -0,0 +1,66 @@
+[
+  "ajar"
+  "alias"
+  "as"
+  "bits"
+  "closed"
+  "compose"
+  "const"
+  "enum"
+  "error"
+  "flexible"
+  "library"
+  "open"
+  ; "optional" we did not specify a node for optional yet
+  "overlay"
+  "protocol"
+  "reserved"
+  "resource"
+  "service"
+  "strict"
+  "struct"
+  "table"
+  "type"
+  "union"
+  "using"
+] @keyword
+
+(primitives_type) @type
+
+(builtin_complex_type) @type
+
+(const_declaration
+  (identifier) @constant)
+
+[
+  "="
+  "|"
+  "&"
+  "->"
+] @operator
+
+(attribute
+  (identifier) @attribute)
+
+(attribute
+  "@" @attribute)
```

Comment:
If there's only 1 identifier for an attribute, consider grouping these 2 into one
```suggestion
(attribute
  "@" @attribute
  (identifier) @attribute)
```